creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -249,7 +249,7 @@ async function _handleIssueSyncRequest({
         syncType,
         syncMode: SYNC_MODES.SyncSecondaryFromPrimary,
         syncRequestParameters,
-        attemptNumber
+        attemptNumber: attemptNumber + 1
```

Comment:
hmm so this was actually intentional to have the same attemptNumber here because it gets incremented above [here](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts#L123). but this is showing fewer syncs in your tests?